src/components/paragraphs/sup-file-list/file-list-selection.tsx

```diff
@@ -31,8 +31,8 @@ const FileListSelection = ({fileOptions, label, ...props}: Props) => {
         />
       </div>
       {chosenItem && (
-        <Button href={chosenItem.url} prefetch={false} className="flex items-center gap-5">
-          Download {chosenItem.label} <DocumentArrowDownIcon width={30} />
+        <Button href={chosenItem.url} prefetch={false} className="!rs-py-1 !rs-px-2 !type-0 flex items-center gap-10">
```

Comment:
You need the important `!`?

Reply:
For some reason the className styles appended at the end of the existing button styles aren't overriding unless I add the important `!`. If we prefer not to use `!`, I can look further into it in a bit 😃

Reply:
The reason is that tailwind does not know what to do with the `rs-` classes since they aren't standard tailwind classes. The twMerge doesn't know what it should replace or change. If you can use standard tailwind classes, it might work as expected.